### ISSUE TITLE
Onenote Update Disables Part of the 'Customize Section Color' Option

### ISSUE THREAD

**Issue by PaulJamesTaylor** [stevencohn/OneMore#1293]
# How can I help you?

I use the Onemore option to change the section color by right clicking on the section tab and selecting 'customize section color'.  I have gotten in the habit of this because I find the default pastel section colors applied a little annoying so I change them to grey.  I should note that its not so much the color of the tab that is annoying but rather the section on the side where the pages are listed.

However, today my company pushed out an update for office which changed all of my section color tabs to a light purple and the side page panel to white.  In regards to the tabs its no big deal, I can apply the custom section color to the tabs back to grey, but the problem is that unlike the prior version this does not change the color of the page section on the side.  Now for all sections this is pure glaring white, which again is rather annoying.

Is there any way to change the color of the page section using Onemore?  I'm on Office 365 version 2308.  I'm thinking this probably isn't available in the API but am curious to confirm that.

THE FINE PRINT This is a hobby project that I started for myself. I'm happy to take requests and will promise to
consider each one carefully but offer no guarantees that I will ultimately agree to their
usefulness or have time to implement any of them. If OneNote offers a "close enough" work-around
then I will probably reject the request - you've been warned!

**Comment by stevenmcohn** [comment]
Don't think there's anything I can do via the API. But are you sure the pushed Office update didn't reset your Office theme back to White or Colorful? Did you set the theme to Black previously?

You can check the them in File/Options/General, should be at the bottom of that page.

**Comment by stevencohn** [comment]
Don't think there's anything I can do via the API. But are you sure the pushed Office update didn't reset your Office theme back to White or Colorful? Did you set the theme to Black previously?

You can check the them in File/Options/General, should be at the bottom of that page.

**Comment by PaulJamesTaylor** [comment]
@stevencohn  I have always always kept the office theme set to White and that hasn't changed. It looks like they decoupled the side page coloring from the tab color.  No matter what color you apply to the tab it doesn't affect the side panel color which always remains white.  See screenshot below for example.  

This is new, the side page used to be a variant of the tab color.  For example, if I set the color of the section to 226,222,201 the side panel (according to ColorPixel) would be 245,244,237.  Now its always 255,255,255 no matter what I set the section color.  A pretty terrible regression in my opinion.

![image](https://github.com/stevencohn/OneMore/assets/22547745/9d2514e0-0cb8-451d-87c7-4d0f80824b50)
